<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgSimpleFilter_scaleCols</class>
 <widget class="QDialog" name="CQDlgSimpleFilter_scaleCols">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Intensity Scaling</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QDialogButtonBox" name="qqOkCancel">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>370</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Colors specified as</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QRadioButton" name="qqRgb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>82</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>RGB</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="qqHsl">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>HSL</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Pre-offset</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Red/hue</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqRed1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqGreen1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Green/saturation</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Blue/luminosity</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqBlue1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Scaling</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Red/hue</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqRed2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqGreen2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Green/saturation</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Blue/luminosity</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqBlue2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Post-offset</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Red/hue</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqRed3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqGreen3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Green/saturation</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Blue/luminosity</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqBlue3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqRgb</tabstop>
  <tabstop>qqHsl</tabstop>
  <tabstop>qqRed1</tabstop>
  <tabstop>qqGreen1</tabstop>
  <tabstop>qqBlue1</tabstop>
  <tabstop>qqRed2</tabstop>
  <tabstop>qqGreen2</tabstop>
  <tabstop>qqBlue2</tabstop>
  <tabstop>qqRed3</tabstop>
  <tabstop>qqGreen3</tabstop>
  <tabstop>qqBlue3</tabstop>
  <tabstop>qqOkCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
